### DISCUSSION TITLE
conio. h no such file or directory

### DISCUSSION THREAD

**Discussion by iamhamza112** [microsoft/vscode-cpptools#9234] [Q&A]
I am getting this error in vscode. OS is ubuntu 20.04. vs code version is 1.66.2
![image](https://user-images.githubusercontent.com/61885781/165035308-de0d5fdb-cd12-43d3-998d-3ce7cc762012.png)

**Comment by sean-mcmanus** [comment]
`conio.h` is not a part of the standard C library, so it's not available with gcc -- see https://en.wikipedia.org/wiki/Conio.h .
